src/sentry/api/serializers/models/sentry_app.py

```diff
@@ -53,4 +53,15 @@ def serialize(self, obj, attrs, user, access):
                 }
             )
 
+        for img in SentryAppAvatar.objects.filter(sentry_app=obj):
+            data.update(
+                {
+                    "avatar": {
+                        "avatarType": img.avatar_type,
+                        "avatarUuid": img.ident,
+                        "color": img.color,
+                    }
+                }
+            )
```

Comment:
Just noticed, I dont know if this will return all the SentryAppAvatars or just the last one in the `filter`, since the field `avatar` holds a dict, rather than an array of dicts. Haven't tested this, just wondering 👍 

```python
data.update(
    {
        avatars: [
            {
                "avatarType": img.avatar_type,
                "avatarUuid": img.ident,
                "color": img.color,
            }
            for img in SentryAppAvatar.objects.filter(sentry_app=obj)
        ]
    }
)
```